acts_as_archival.gemspec

```diff
@@ -33,6 +33,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "rake"
   gem.add_development_dependency "rr"
   gem.add_development_dependency "sqlite3"
+  gem.add_development_dependency "rubocop", "~> 0.47.1"
```

Comment:
If there's not a good reason to lock this version, let's remove the requirement.